tests/eval/control.joke

```diff
@@ -0,0 +1,443 @@
+;   Copyright (c) Rich Hickey. All rights reserved.
+;   The use and distribution terms for this software are covered by the
+;   Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+;   which can be found in the file epl-v10.html at the root of this distribution.
+;   By using this software in any fashion, you are agreeing to be bound by
+;   the terms of this license.
+;   You must not remove this notice, or any other, from this software.
+
+; Author: Frantisek Sodomka, Mike Hinchey, Stuart Halloway
+
+;;
+;;  Test "flow control" constructs.
+;;
+
+(ns joker.test-clojure.control
+  (:require [joker.test :refer [deftest is are testing]]
+            [tests.test_helper :refer :all]))
```

Comment:
Since Joker, unlike Clojure, doesn't convert dashes to underscores when loading files, let's rename `test_helper` (the file and the namespace) to `test-helper`. Also, please don't use `:refer :all` as it makes the code unlintable.

Reply:
Done.